BL33_AP_UEFI.fd

```diff
@@ -0,0 +1 @@
+/home/victor.chong/work/swg/repo_960/build/../edk2/Build/HiKey960/RELEASE_GCC49/FV/BL33_AP_UEFI.fd
```

Comment:
seems wrong

Reply:
Oops sorry! This got added in by accident. Will push a new commit.

Reply:
File removed. Thanks for noticing!